test/testinput/gnssro_wmoBUFR2ioda.yaml

```diff
@@ -48,170 +48,151 @@ observations:
             mnemonic: SAID
           instrumentIdentifier:
             mnemonic: SIID
-          satelliteClassification:
-            mnemonic: SCLF
           platformTransmitterId:
             mnemonic: PTID
-          originatingCenter:
+          dataProviderOrigin:
             mnemonic: OGCE
           qualityFlags:
             mnemonic: QFRO
           percentConfidence:
             mnemonic: PCCF
-          impactParameter:
+          impactParameterRO:
             mnemonic: IMPP
           geoidUndulation:
             mnemonic: GEODU
-          azimuth:
+          sensorAzimuthAngle:
             mnemonic: BEARAZ
           earthRadiusCurvature:
             mnemonic: ELRC
 
           bendingAngle:
             mnemonic: BNDA
-          refractivity:
+          atmosphericRefractivity:
             mnemonic: ARFR
 
     ioda:
       backend: netcdf
       obsdataout: "./testrun/gnssro_2020-306-2358C2E6.nc"
 
       dimensions:
-        - name: "nlocs"
+        - name: "Location"
           size: variables/latitude.nrows
 
       variables:
         - name: "MetaData/dateTime"
           source: variables/timestamp
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "dateTime"
           units: "seconds since 1970-01-01T00:00:00Z"
 
         - name: "MetaData/latitude"
           source: variables/latitude
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Latitude"
           units: "degrees_north"
 
         - name: "MetaData/longitude"
           source: variables/longitude
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Longitude"
           units: "degrees_east"
 
         - name: "MetaData/height"
           source: variables/height
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Height"
           units: "m"
 
         - name: "MetaData/geopotentialHeight"
           source: variables/geopotentialHeight
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Geopotential height"
           units: "m"
 
-        - name: "MetaData/airPressure"
-          coordinates: "longitude latitude"
-          source: variables/airPressure
-          dimensions: [ "nlocs" ]
+        - name: "MetaData/pressure"
```

Comment:
I know this is not really part of the PR, but why is this changed? I don't like this. What about `water pressure` or `vapor pressure`. `pressure` is too ambiguous IMO.

Reply:
> I know this is not really part of the PR, but why is this changed? I don't like this. What about `water pressure` or `vapor pressure`. `pressure` is too ambiguous IMO.


I could not agree more!  This is a definite issue/problem, but when I reviewed the choices in WMO bufr tables, there really isn't anything separated for ```pressure```.  I debated just diverting from BUFR tables, but didn't.  We can still diverge if we want.  But, at this point, I would need to change another 10-15 files (again).

Reply:
maybe something to discuss today with Ryan/the group? I'm fine with leaving as is for now, but really think airPressure is much better than pressure, especially since the long name is still 'air pressure'